packages/sourcing/test/fixtures/fileManagers.fixture.ts

```diff
@@ -0,0 +1,10 @@
+
+import { FileManager } from '../../src';
+
+import TestFileSystem from './TestFileSystem';
+import PATHS from './paths.fixture';
+
+const fileSystem = new TestFileSystem(PATHS.CONFIGS.ROOT_PATH);
+const fileManager = new FileManager(PATHS.CONFIGS.BASE_LOCATION, fileSystem);
+
+export { fileManager };
```

Comment:
This file now only returns a single file manager. Therefore we can either:

a) rename the file to `fileManager.fixture.ts`
b) add the file manager to an extendable object.